src/libPMacc/include/functor/Filtered.hpp

```diff
@@ -110,8 +109,10 @@ namespace acc
 
     /** specialization of Filtered (with unary filter)
      *
-     * This specialization can only used if T_Filter is of the type PMacc::filter::Interface
-     * and T_Functor is of the type PMacc::filter::Interface
+     * This specialization can only be used if T_Filter is of the type PMacc::filter::Interface
+     * and T_Functor is of the type PMacc::functor::Interface.
+     * An unary filters means that each argument can only pass the same filter
```

Comment:
nope, an because u is a vowel.

Filters are functors with boolean return value. An unary filter is an unary functor. That means if you want to run a functor on several arguments, an unary filter can only pass one argument after the other and chain the results to an other bool which then decides, if the (then filtered) functor can be executed.

Admittedly, also took me a while...